-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ce8476/35080/summary.html
**COMMIT**: 3a78801fd988ab23c0aa668a1cc9d3c47c5d1689
**CMSSW**: CMSSW_13_3_X_2023-10-06-1100/el8_amd64_gcc11
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/42958/35080/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Done generating edm plugin poisoned information
gmake[1]: *** [config/SCRAM/GMake/Makefile.rules:1937: CompilePython] Error 1
>> Plugins of all types refreshed.
gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_X_2023-10-06-1100'
gmake: *** [config/SCRAM/GMake/Makefile.rules:1825: src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
+ eval scram build outputlog '&&' '(python3' /data/cmsbld/jenkins/workspace/ib-run-pr-tests/cms-bot/buildLogAnalyzer.py --logDir /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_X_2023-10-06-1100/tmp/el8_amd64_gcc11/cache/log/src '||' 'true)'
++ scram build outputlog
>> Entering Package Configuration/AlCa
</pre>

--- a/Configuration/AlCa/python/autoCond.py
+++ b/Configuration/AlCa/python/autoCond.py
@@ -2,93 +2,95 @@ autoCond = {
 
     ### NEW KEYS ###
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run1
-    'run1_design'                  : '131X_mcRun1_design_v2',
+    'run1_design'                  :    '131X_mcRun1_design_v2',
     # GlobalTag for MC production (pp collisions) with realistic alignment and calibrations for Run1
-    'run1_mc'                      : '131X_mcRun1_realistic_v2',
+    'run1_mc'                      :    '131X_mcRun1_realistic_v2',
     # GlobalTag for MC production (Heavy Ions collisions) with realistic alignment and calibrations for Run1
-    'run1_mc_hi'                   : '131X_mcRun1_HeavyIon_v2',
+    'run1_mc_hi'                   :    '131X_mcRun1_HeavyIon_v2',
     # GlobalTag for MC production with pessimistic alignment and calibrations for Run2
-    'run2_mc_50ns'                 : '131X_mcRun2_startup_v2',
+    'run2_mc_50ns'                 :    '131X_mcRun2_startup_v2',
     # GlobalTag for MC production (2015 L1 Trigger Stage1) with startup-like alignment and calibrations for Run2, L1 trigger in Stage1 mode
-    'run2_mc_l1stage1'             : '131X_mcRun2_asymptotic_l1stage1_v2',
+    'run2_mc_l1stage1'             :    '131X_mcRun2_asymptotic_l1stage1_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'                  : '131X_mcRun2_design_v2',
+    'run2_design'                  :    '131X_mcRun2_design_v2',
     #GlobalTag for MC production with optimistic alignment and calibrations for 2016, prior to VFP change
-    'run2_mc_pre_vfp'              : '131X_mcRun2_asymptotic_preVFP_v2',
+    'run2_mc_pre_vfp'              :    '131X_mcRun2_asymptotic_preVFP_v2',
     #GlobalTag for MC production with optimistic alignment and calibrations for 2016, after VFP change
-    'run2_mc'                      : '131X_mcRun2_asymptotic_v2',
+    'run2_mc'                      :    '131X_mcRun2_asymptotic_v2',
     # GlobalTag for MC production (cosmics) with starup-like alignment and calibrations for Run2, Strip tracker in peak mode
-    'run2_mc_cosmics'              : '131X_mcRun2cosmics_asymptotic_deco_v2',
+    'run2_mc_cosmics'              :    '131X_mcRun2cosmics_asymptotic_deco_v2',
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
-    'run2_mc_hi'                   : '131X_mcRun2_HeavyIon_v2',
+    'run2_mc_hi'                   :    '131X_mcRun2_HeavyIon_v2',
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run2
-    'run2_mc_pa'                   : '131X_mcRun2_pA_v2',
+    'run2_mc_pa'                   :    '131X_mcRun2_pA_v2',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'                    : '124X_dataRun2_v2',
+    'run2_data'                    :    '124X_dataRun2_v2',
     # GlobalTag for Run2 data 2018B relvals only: HEM-15-16 fail
-    'run2_data_HEfail'             : '124X_dataRun2_HEfail_v2',
+    'run2_data_HEfail'             :    '124X_dataRun2_HEfail_v2',
     # GlobalTag for Run2 HI data
-    'run2_data_promptlike_hi'      : '124X_dataRun2_PromptLike_HI_v1',
+    'run2_data_promptlike_hi'      :    '124X_dataRun2_PromptLike_HI_v1',
     # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'              : '123X_dataRun2_HLT_relval_v3',
-    # GlobalTag for Run3 HLT: identical to the online GT (132X_dataRun3_HLT_v2) but with snapshot at 2023-08-11 17:35:000 (UTC)
-    'run3_hlt'                     : '132X_dataRun3_HLT_frozen_v1',
-    # GlobalTag for Run3 data relvals (express GT) - identical to 132X_dataRun3_Express_v2 but with snapshot at 2023-08-11 17:35:00 (UTC)
-    'run3_data_express'            : '132X_dataRun3_Express_frozen_v1',
-    # GlobalTag for Run3 data relvals (prompt GT) - identical to 132X_dataRun3_Prompt_v2 but with snapshot at 2023-08-11 17:35:00 (UTC)
-    'run3_data_prompt'             : '132X_dataRun3_Prompt_frozen_v1',
-    # GlobalTag for Run3 offline data reprocessing - snapshot at 2023-08-11 17:04:34 (UTC)
-    'run3_data'                    : '132X_dataRun3_v1',
+    'run2_hlt_relval'              :    '123X_dataRun2_HLT_relval_v3',
+    # GlobalTag for Run3 HLT: identical to the online GT (132X_dataRun3_HLT_v2) but with snapshot at 2023-10-04 21:27:37 (UTC)
+    'run3_hlt'                     :    '133X_dataRun3_HLT_frozen_v1',
+    # GlobalTag for Run3 data relvals (express GT) - 132X_dataRun3_Express_v4 with Ecal CC timing tags and snapshot at 2023-10-04 21:27:37 (UTC)
+    'run3_data_express'            :    '133X_dataRun3_Express_frozen_v1',
+    # GlobalTag for Run3 data relvals (prompt GT) - 132X_dataRun3_Prompt_v4 with Ecal CC timing tags and snapshot at 2023-10-04 21:27:37 (UTC)
+    'run3_data_prompt'             :    '133X_dataRun3_Prompt_frozen_v1',
+    # GlobalTag for Run3 offline data reprocessing - snapshot at 2023-10-06 09:32:44 (UTC)
+    'run3_data'                    :    '133X_dataRun3_v2',
+    # GlobalTag for Run3 offline data reprocessing with Prompt GT, currenlty for 2022FG - snapshot at 2023-10-06 10:12:57 (UTC)
+    'run3_data_PromptAnalysis'     :    '133X_dataRun3_PromptAnalysis_v1',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2017 (and 0,0,~0-centred beamspot)
-    'phase1_2017_design'           : '131X_mc2017_design_v2',
+    'phase1_2017_design'           :    '131X_mc2017_design_v2',
     # GlobalTag for MC production with realistic conditions for Phase1 2017 detector
-    'phase1_2017_realistic'        : '131X_mc2017_realistic_v2',
+    'phase1_2017_realistic'        :    '131X_mc2017_realistic_v2',
     # GlobalTag for MC production (cosmics) with realistic alignment and calibrations for Phase1 2017 detector, Strip tracker in DECO mode
-    'phase1_2017_cosmics'          : '131X_mc2017cosmics_realistic_deco_v2',
+    'phase1_2017_cosmics'          :    '131X_mc2017cosmics_realistic_deco_v2',
     # GlobalTag for MC production (cosmics) with realistic alignment and calibrations for Phase1 2017 detector, Strip tracker in PEAK mode
-    'phase1_2017_cosmics_peak'     : '131X_mc2017cosmics_realistic_peak_v2',
+    'phase1_2017_cosmics_peak'     :    '131X_mc2017cosmics_realistic_peak_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for full Phase1 2018 (and 0,0,0-centred beamspot)
-    'phase1_2018_design'           : '131X_upgrade2018_design_v2',
+    'phase1_2018_design'           :    '131X_upgrade2018_design_v2',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector
-    'phase1_2018_realistic'        : '131X_upgrade2018_realistic_v2',
+    'phase1_2018_realistic'        :    '131X_upgrade2018_realistic_v2',
     # GlobalTag for MC production with realistic run-dependent (RD) conditions for full Phase1 2018 detector
-    'phase1_2018_realistic_rd'     : '131X_upgrade2018_realistic_RD_v2',
+    'phase1_2018_realistic_rd'     :    '131X_upgrade2018_realistic_RD_v2',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector for Heavy Ion
-    'phase1_2018_realistic_hi'     : '131X_upgrade2018_realistic_HI_v2',
+    'phase1_2018_realistic_hi'     :    '131X_upgrade2018_realistic_HI_v2',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector: HEM-15-16 fail
-    'phase1_2018_realistic_HEfail' : '131X_upgrade2018_realistic_HEfail_v3',
+    'phase1_2018_realistic_HEfail' :    '131X_upgrade2018_realistic_HEfail_v3',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in DECO mode
-    'phase1_2018_cosmics'          : '131X_upgrade2018cosmics_realistic_deco_v2',
+    'phase1_2018_cosmics'          :    '131X_upgrade2018cosmics_realistic_deco_v2',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
-    'phase1_2018_cosmics_peak'     : '131X_upgrade2018cosmics_realistic_peak_v3',
+    'phase1_2018_cosmics_peak'     :    '131X_upgrade2018cosmics_realistic_peak_v3',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2022
-    'phase1_2022_design'           : '132X_mcRun3_2022_design_v1',
+    'phase1_2022_design'           :    '133X_mcRun3_2022_design_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2022
-    'phase1_2022_realistic'        : '132X_mcRun3_2022_realistic_v2',
+    'phase1_2022_realistic'        :    '133X_mcRun3_2022_realistic_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2022 post-EE+ leak
-    'phase1_2022_realistic_postEE' : '132X_mcRun3_2022_realistic_postEE_v2',
+    'phase1_2022_realistic_postEE' :    '133X_mcRun3_2022_realistic_postEE_v1',
     # GlobalTag for MC production (cosmics) with realistic conditions for Phase1 2022,  Strip tracker in DECO mode
-    'phase1_2022_cosmics'          : '132X_mcRun3_2022cosmics_realistic_deco_v2',
+    'phase1_2022_cosmics'          :    '133X_mcRun3_2022cosmics_realistic_deco_v1',
     # GlobalTag for MC production (cosmics) with perfectly aligned and calibrated detector for Phase1 2022, Strip tracker in DECO mode
-    'phase1_2022_cosmics_design'   : '132X_mcRun3_2022cosmics_design_deco_v1',
+    'phase1_2022_cosmics_design'   :    '133X_mcRun3_2022cosmics_design_deco_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2022 detector for Heavy Ion
-    'phase1_2022_realistic_hi'     : '132X_mcRun3_2022_realistic_HI_v1',
+    'phase1_2022_realistic_hi'     :    '133X_mcRun3_2022_realistic_HI_v1',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2023
-    'phase1_2023_design'           : '132X_mcRun3_2023_design_v1',
+    'phase1_2023_design'           :    '133X_mcRun3_2023_design_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2023
-    'phase1_2023_realistic'        : '132X_mcRun3_2023_realistic_v4',
+    'phase1_2023_realistic'        :    '133X_mcRun3_2023_realistic_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 post BPix issue 2023
-    'phase1_2023_realistic_postBPix'  : '132X_mcRun3_2023_realistic_postBPix_v3',
+    'phase1_2023_realistic_postBPix'  : '133X_mcRun3_2023_realistic_postBPix_v1',
     # GlobalTag for MC production (cosmics) with realistic conditions for Phase1 2023,  Strip tracker in DECO mode
-    'phase1_2023_cosmics'          : '132X_mcRun3_2023cosmics_realistic_deco_v3',
+    'phase1_2023_cosmics'          :    '133X_mcRun3_2023cosmics_realistic_deco_v1',
     # GlobalTag for MC production (cosmics) with perfectly aligned and calibrated detector for Phase1 2023, Strip tracker in DECO mode
-    'phase1_2023_cosmics_design'   : '132X_mcRun3_2023cosmics_design_deco_v1',
+    'phase1_2023_cosmics_design'   :    '133X_mcRun3_2023cosmics_design_deco_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2023 detector for Heavy Ion
-    'phase1_2023_realistic_hi'     : '132X_mcRun3_2023_realistic_HI_v2',
+    'phase1_2023_realistic_hi'     :    '133X_mcRun3_2023_realistic_HI_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2024
-    'phase1_2024_realistic'        : '132X_mcRun3_2024_realistic_v2',
+    'phase1_2024_realistic'        :    '133X_mcRun3_2024_realistic_v1',
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'             : '132X_mcRun4_realistic_v2'
+    'phase2_realistic'             :    '133X_mcRun4_realistic_v1'
 }
 
 aliases = {

